### DISCUSSION TITLE
Still havent received my Student Pack

### DISCUSSION THREAD

**Discussion by AlpMeteSenel** [community/community#136710] [GitHub Education] [answered]
### Hi everyone,

Hello,
Even though it's been few days since I got verified, I still haven't received my Student benefits yet. Could anyone help?

**Comment by davevad93** [comment, marked as answer]
Hi @AlpMeteSenel ,

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @AlpMeteSenel was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks; 

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by AlpMeteSenel** [reply]
It's funny to see people who created a new topic after me getting their benefits before me..
